### DISCUSSION TITLE
Flatpak enabled by default

### DISCUSSION THREAD

**Discussion by 2000Maxx** [elementary/triage#374] [Ideas]
Dont know if this is already being adapted but as Flatpaks seem to be the way to go, they should be enabled by default.

Best regards

**Comment by tintou** [comment]
It is the case since a while now https://blog.elementary.io/elementary-appcenter-flatpak/

**Reply by 2000Maxx** [reply]
Ah, sorry. I did misworded it. I meant Flathub. I understand the philosophy of prefering native apps, however many users will likely need additional apps and having those enabled by default or at least an intuitive way towards enabling Flathub would be nice to have.
Thanks for the reply :)
